openshift_tools/python-openshift-tools.spec

```diff
@@ -38,27 +38,109 @@ cp -p web/*.py %{buildroot}%{python_sitelib}/openshift_tools/web
+
+
+
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-zagg subpackage
+# ----------------------------------------------------------------------------------
+%package monitoring-zagg
+Summary:       OpenShift Tools Zagg Python Libraries Package
+Requires:      python2,python-openshift-tools,python-openshift-tools-web,python-zbxsend
+BuildArch:     noarch
+
+%description monitoring-zagg
+Zagg Python libraries developed for monitoring OpenShift.
+
+%files monitoring-zagg
+%{python_sitelib}/openshift_tools/monitoring/metricmanager.py
+%{python_sitelib}/openshift_tools/monitoring/metricmanager.py[co]
+%{python_sitelib}/openshift_tools/monitoring/zagg*.py
+%{python_sitelib}/openshift_tools/monitoring/zagg*.py[co]
+%{python_sitelib}/openshift_tools/monitoring/zabbix_metric_processor.py
+%{python_sitelib}/openshift_tools/monitoring/zabbix_metric_processor.py[co]
+
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-zabbix subpackage
+# ----------------------------------------------------------------------------------
+%package monitoring-zabbix
+Summary:       OpenShift Tools Zagg Python Libraries Package
+Requires:      python2,python-openshift-tools
+BuildArch:     noarch
+
+%description monitoring-zabbix
+Zagg Python libraries developed for monitoring OpenShift.
+
+%files monitoring-zabbix
+%{python_sitelib}/openshift_tools/monitoring/zbxapi.py
+%{python_sitelib}/openshift_tools/monitoring/zbxapi.py[co]
+
+
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-aws subpackage
+# ----------------------------------------------------------------------------------
+%package monitoring-aws
+Summary:       OpenShift Tools AWS Python Libraries Package
+Requires:      python2,python-openshift-tools,python-awscli
+BuildArch:     noarch
+
+%description monitoring-aws
+AWS Python libraries developed for monitoring OpenShift.
+
+%files monitoring-aws
+%{python_sitelib}/openshift_tools/monitoring/awsutil.py
+%{python_sitelib}/openshift_tools/monitoring/awsutil.py[co]
+
+
+# ----------------------------------------------------------------------------------
+# python-openshift-tools-monitoring-openshift subpackage
+# ----------------------------------------------------------------------------------
+%package monitoring-openshift
+Summary:       OpenShift Tools Openshift Python Libraries Package
+Requires:      python2,python-openshift-tools,atomic-openshift
+BuildArch:     noarch
+
+%description monitoring-openshift
+Openshift Python libraries developed for monitoring OpenShift.
+
+%files monitoring-openshift
+%{python_sitelib}/openshift_tools/monitoring/ocutil.py
+%{python_sitelib}/openshift_tools/monitoring/ocutil.py[co]
 
 
 # ----------------------------------------------------------------------------------
```

Comment:
python-openshift-tools-ansible subpackage needs to depend on python-openshift-tools-monitoring-zabbix.